# Name of the objec
string object_name

bool use_only_local_resources

# RMP
temoto_core/RMPRequest trr

---

# Object msg that contains all information about
# the requested object
temoto_context_manager/ObjectContainer object

# Topic where the information about the required
# object is going to be published
string object_topic

# RMP
temoto_core/RMPResponse trr
